generator.py

```diff
@@ -147,6 +147,9 @@ def main():
     if conf.gir_files_path == DEFAULT_GIR_FILES_DIRECTORY:
         if def_check_submodule(conf.gir_files_path, conf) == FAILURE:
             return 1
+    elif not isdir(conf.gir_files_path):
+        print("`{}` dir doesn't exist. Aborting...".format(conf.gir_files_path))
+        return 1
```

Comment:
Careful, this is dealt with by `def directory_path(path):` above, and should otherwise be `conf.gir_files_path.is_dir()`.

Reply:
Arf! Removing useless code I added back.